lib/udf_findhotel_utils.rb

```diff
@@ -89,18 +89,22 @@ class UdfFindhotelUtils
           type:        :function,
           name:        :make_bing_click_batch_id,
           description: "Returns a unique identifier for a batch of clicks reported by Bing.",
-          params:      "ad_group_id varchar(max), ad_id varchar(max), keyword varchar(max), device_type varchar(max), network varchar(max), bid_match_type varchar(max)",
+          params:      "ad_group_id varchar(max), ad_id varchar(max), keyword_id varchar(max), device_type varchar(max), network varchar(max), bid_match_type varchar(max)",
           return_type: "varchar(max)",
           body:        %~
             import hashlib
             import json
 
+            actual_network = "Owned" if network in ("Bing and Yahoo! search", "AOL search") else \
```

Comment:
I think a better condition is:

```
actual_network = "Syndicated" if network=="Syndicated search partners" else "Owned"
```

The reason is that Bing groups under "Syndicated" anything that is not owned by Microsoft/Bing. But if Bing would acquire a new advertising network we may get a new value (e.g. "Cool MS owned network") which should be regarded as "Owned".